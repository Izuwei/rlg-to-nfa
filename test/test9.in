A,B,C,D
a,b,c
A
A->abcB
B->aB
B->#
