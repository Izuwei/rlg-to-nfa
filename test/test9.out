1,2,3,4,5,6
abc
1
4
1,a,2
2,b,3
3,c,4
4,a,4
